.travis.yml

```diff
@@ -27,11 +33,10 @@ before_install:
 
 install:
   - python setup.py -q develop
```

Comment:
try to use the following: `pip install -U -e .`

Reply:
I didn't knew about it earlier. Thanks a lot, finally learnt a way to install dependencies.:smile: 
I have updated the code as per your suggestions.